diff --git a/dist/config/flat-routes.js b/dist/config/flat-routes.js
index 57edc32bf2d370bc59d5540d84591100730ee8f0..b56814c6bc1ea61a3561f843d710f1e096310eaa 100644
--- a/dist/config/flat-routes.js
+++ b/dist/config/flat-routes.js
@@ -282,34 +282,6 @@ function getRouteMap(appDirectory, routePaths, prefix) {
     let routeInfo = routes[i];
     // update parentIds for all routes
     routeInfo.parentId = findParentRouteId(routeInfo, nameMap);
-
-    // remove routes that conflict with other routes
-    let nextRouteInfo = routes[i + 1];
-    if (!nextRouteInfo) continue;
-    let segments = routeInfo.segments;
-    let nextSegments = nextRouteInfo.segments;
-    // if segment count is different, there can't be a conflict
-    if (segments.length !== nextSegments.length) continue;
-    for (let k = 0; k < segments.length; k++) {
-      let segment = segments[k];
-      let nextSegment = nextSegments[k];
-
-      // if segments are different, but they're both dynamic, there's a conflict
-      if (segment !== nextSegment && segment.startsWith(":") && nextSegment.startsWith(":")) {
-        let currentConflicts = conflicts.get(routeInfo.path || "/");
-        // collect conflicts for later reporting, marking the next route as the conflicting route
-        if (!currentConflicts) {
-          conflicts.set(routeInfo.path || "/", [routeInfo, nextRouteInfo]);
-        } else {
-          currentConflicts.push(nextRouteInfo);
-          conflicts.set(routeInfo.path || "/", currentConflicts);
-        }
-
-        // remove conflicting route
-        routeMap.delete(nextRouteInfo.id);
-        continue;
-      }
-    }
   }
 
   // report conflicts