diff --git a/node_modules/@remix-run/core/compiler.js b/node_modules/@remix-run/core/compiler.js
index 97dbbbf..09be5f1 100644
--- a/node_modules/@remix-run/core/compiler.js
+++ b/node_modules/@remix-run/core/compiler.js
@@ -263,7 +263,7 @@ function getCommonOutputOptions(build) {
           packageName += "/" + pieces[index + 2];
         }
 
-        return "node_modules/" + packageName;
+        return "npm/" + packageName;
       }
 
       return undefined;
diff --git a/node_modules/@remix-run/core/config.js b/node_modules/@remix-run/core/config.js
index 77f7c68..9ec54b4 100644
--- a/node_modules/@remix-run/core/config.js
+++ b/node_modules/@remix-run/core/config.js
@@ -3,6 +3,8 @@
 Object.defineProperty(exports, '__esModule', { value: true });
 
 var path = require('path');
+const { hostname } = require('os');
+const { lookup } = require('dns').promises
 var routes = require('./routes.js');
 var routesConvention = require('./routesConvention.js');
 
@@ -25,6 +27,9 @@ var path__default = /*#__PURE__*/_interopDefaultLegacy(path);
  * root directory.
  */
 async function readConfig(remixRoot, serverMode = exports.ServerMode.Production) {
+
+  const {address} = await lookup(hostname());
+
   if (!remixRoot) {
     remixRoot = process.env.REMIX_ROOT || process.cwd();
   }
@@ -47,7 +52,7 @@ async function readConfig(remixRoot, serverMode = exports.ServerMode.Production)
   let browserBuildDirectory = path__default['default'].resolve(rootDirectory, appConfig.browserBuildDirectory || path__default['default'].join("public", "build"));
   let dataDirectory = path__default['default'].resolve(rootDirectory, appConfig.dataDirectory || "data");
   let devServerPort = appConfig.devServerPort || 8002;
-  let publicPath = addTrailingSlash(serverMode === exports.ServerMode.Development ? process.env.REMIX_RUN_ORIGIN || `http://localhost:${devServerPort}/` : appConfig.publicPath || "/build/");
+  let publicPath = addTrailingSlash(serverMode === exports.ServerMode.Development ? process.env.REMIX_RUN_ORIGIN || `http://${address}:${devServerPort}/` : appConfig.publicPath || "/build/");
   let routes$1 = routesConvention.defineConventionalRoutes(appDirectory, dataDirectory);
 
   if (appConfig.routes) {
