diff --git a/node_modules/@remix-run/dev/dist/config/flat-routes.js b/node_modules/@remix-run/dev/dist/config/flat-routes.js
index 57edc32..3f563c1 100644
--- a/node_modules/@remix-run/dev/dist/config/flat-routes.js
+++ b/node_modules/@remix-run/dev/dist/config/flat-routes.js
@@ -14,6 +14,7 @@ Object.defineProperty(exports, '__esModule', { value: true });
 
 var path = require('node:path');
 var glob = require('fast-glob');
+
 var routes = require('./routes.js');
 var routesConvention = require('./routesConvention.js');
 
@@ -278,38 +279,9 @@ function getRouteMap(appDirectory, routePaths, prefix) {
   let routes = Array.from(routeMap.values()).sort((a, b) => {
     return b.segments.length - a.segments.length;
   });
-  for (let i = 0; i < routes.length; i++) {
-    let routeInfo = routes[i];
+  for (let routeInfo of routes) {
     // update parentIds for all routes
     routeInfo.parentId = findParentRouteId(routeInfo, nameMap);
-
-    // remove routes that conflict with other routes
-    let nextRouteInfo = routes[i + 1];
-    if (!nextRouteInfo) continue;
-    let segments = routeInfo.segments;
-    let nextSegments = nextRouteInfo.segments;
-    // if segment count is different, there can't be a conflict
-    if (segments.length !== nextSegments.length) continue;
-    for (let k = 0; k < segments.length; k++) {
-      let segment = segments[k];
-      let nextSegment = nextSegments[k];
-
-      // if segments are different, but they're both dynamic, there's a conflict
-      if (segment !== nextSegment && segment.startsWith(":") && nextSegment.startsWith(":")) {
-        let currentConflicts = conflicts.get(routeInfo.path || "/");
-        // collect conflicts for later reporting, marking the next route as the conflicting route
-        if (!currentConflicts) {
-          conflicts.set(routeInfo.path || "/", [routeInfo, nextRouteInfo]);
-        } else {
-          currentConflicts.push(nextRouteInfo);
-          conflicts.set(routeInfo.path || "/", currentConflicts);
-        }
-
-        // remove conflicting route
-        routeMap.delete(nextRouteInfo.id);
-        continue;
-      }
-    }
   }
 
   // report conflicts
